source/scss/style.scss

```diff
@@ -26,4 +26,8 @@
 .chart-placeholder ul {
     list-style-type: none;
     font-size:0.6em;
-}
\ No newline at end of file
+}
+
+input:disabled+label {
+  color: #d3d3d3;
```

Comment:
I think the new color is very noticeable. Thanks!
![greyed-out](https://user-images.githubusercontent.com/13045020/33846327-60c6016e-de74-11e7-8306-9ddf0942d3ed.png)